src/main.cpp

```diff
@@ -23,24 +39,26 @@ int main() {
         .max_threads(c.get_max_threads())
         .digest_auth();
 
-    hello_world_resource hwr;
-    ws.register_resource("/hello", &hwr);
+    spdlog::info("Server started with settings:");
+    c.log_configs();
 
-    std::cout << "\nSERVER START" << std::endl;
-    std::cout << c.toString() << std::endl;
-    std::cout << "Ongoing Status: " << std::endl;
+    //Monitoring m("log.txt");
+    //std::thread monitor_thread(start_monitoring, std::ref(m));
```

Comment:
Although it's commented out, you could do `std::thread monitor_thread(&Monitoring::start, std::ref(m));` to get rid of the extra function.

Reply:
I'll look into this once I expand that live monitoring.